src/app/state/todos/todos.reducer.ts

```diff
@@ -1,21 +1,45 @@
-import { ADD_TODO, REMOVE_TODO, COMPLETE_TODO } from './todos.types';
+import { ADD_TODO, REMOVE_TODO, COMPLETE_TODO, SELECT_TODO } from './todos.types';
+import { createTodo } from './todos.actions';
 
-export function todosReducer(state = [], action) {
+export const initialState = {
+  todos: [
+    createTodo('First todo item', 'Lorem ipsum sit dolor'),
+    createTodo('Second todo', 'Lorem ipsum sit dolor'),
+    createTodo('Third todo', 'Lorem ipsum sit dolor'),
+    createTodo('Fourth todo', 'Lorem ipsum sit dolor'),
+    createTodo('Fifth todo', 'Lorem ipsum sit dolor')
+  ],
+  selectedTodoId: null
+}
```

Comment:
The other "issue" with attempting to set or "select" a specific todo, is that the todo state was originally a flat array. There is no way to keep track of which todo was selected.

We need to make our todo state a little more structured so that we can support this type of metadata, which also entails updating our reducers a bit

Once again, there's other ways to potentially handle this, and other ways we might simplify our reducers and keep them from getting bloated as this one is starting to do